.github/workflows/import-test.yml

```diff
@@ -100,6 +100,10 @@ jobs:
       run: |
         # Install NeMo Domain
         pip install "${{ steps.nemo-wheel.outputs.DIST_FILE }}[nlp]"
+        git clone https://github.com/NVIDIA/Megatron-LM.git
+        cd Megatron-LM
+        git checkout 0609f27fe8376f17ab65c001d3d8f35cd8175950
```

Comment:
Isn't it easier to always test main branch ? This will be yet another place to keep track of to update commit ids

Reply:
It's a good idea. Let me update it.